index.js

```diff
@@ -52,7 +56,14 @@ exports.parse = function (str) {
 exports.stringify = function (obj, opts) {
 	opts = opts || {};
 
-	var strict = opts.strict !== false;
+	// Default encode to true to maintain previous behavior
+	if (opts.encode === undefined) {
+		opts.encode = true;
```

Comment:
Here you're overwriting the user-supplied object, which means if the user calls this method twice with the same object it will have different behavior. Remember that objects are reference based.